modules/stats.php

```diff
@@ -188,30 +245,61 @@ function stats_get_options() {
 	return $options;
 }
 
+/**
+ * Get Stats Options.
+ *
+ * @access public
+ * @param mixed $option Option.
+ * @return Options|null.
```

Comment:
There are a bunch of these `Options`. Aren't they `array` instead? We don't have a class named `Options`.